### PR TITLE
Add debug utils

### PR DESCRIPTION
**Pull request by ilovesoup**
Add debug utils for various cases.

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tikv-client-lib-java/196)

### CODE REVIEW

File: src/main/java/com/pingcap/tikv/tools/RegionUtils.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class RegionUtils {
+  static public Map<String, Integer>
+  getRegionDistribution(TiSession session, String databaseName, String tableName) {
+    requireNonNull(session, "session is null");
+    requireNonNull(databaseName, "databaseName is null");
+    requireNonNull(tableName, "tableName is null");
```
Comment (Novemser) [inline src/main/java/com/pingcap/tikv/tools/RegionUtils.java:37]: Is it necessary to check string's `isEmpty`?.

File: src/main/java/com/pingcap/tikv/tools/RegionUtils.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    TiTableInfo table = session.getCatalog().getTable(databaseName, tableName);
+    requireNonNull(table, String.format("Table not find %s.%s", databaseName, tableName));
```
Comment (Novemser) [inline src/main/java/com/pingcap/tikv/tools/RegionUtils.java]: Misspell, `not found`.

File: src/main/java/com/pingcap/tikv/tools/RegionUtils.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    ScanBuilder builder = new ScanBuilder();
+    ScanPlan scanPlan = builder.buildScan(ImmutableList.of(), table);
+    List<RegionTask> tasks = RangeSplitter
+        .newSplitter(session.getRegionManager())
+        .splitRangeByRegion(scanPlan.getKeyRanges());
+    HashMap<String, Integer> regionMap = new HashMap<>();
```
Comment (Novemser) [inline src/main/java/com/pingcap/tikv/tools/RegionUtils.java]: Consider using generic `java.util.Map` at left side.

**Review by Novemser — COMMENTED** [review]
Minor.

File: src/main/java/com/pingcap/tikv/tools/RegionUtils.java (hunk 1 above)
Comment (ilovesoup) [reply src/main/java/com/pingcap/tikv/tools/RegionUtils.java:37]: I would postpone it to getTable.

File: src/main/java/com/pingcap/tikv/tools/RegionUtils.java (hunk 3 above)
Comment (ilovesoup) [reply src/main/java/com/pingcap/tikv/tools/RegionUtils.java]: done

File: src/main/java/com/pingcap/tikv/tools/RegionUtils.java (hunk 2 above)
Comment (ilovesoup) [reply src/main/java/com/pingcap/tikv/tools/RegionUtils.java]: done

File: src/main/java/com/pingcap/tikv/tools/RegionUtils.java (hunk 1 above)
Comment (Novemser) [reply src/main/java/com/pingcap/tikv/tools/RegionUtils.java:37]: Okay

**Review by Novemser — APPROVED** [review]
LGTM

**Pull request merged**